src/legacy/core_plugins/tile_map/public/components/wms_interna_options.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { EuiLink, EuiSpacer, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { VisOptionsSetValue } from 'ui/vis/editors/default';
+import { TextInputOption } from '../../../kbn_vislib_vis_types/public/controls/text_input';
+
+export interface TmsLayer {
+  id: string;
+  origin: string;
+  minZoom: string;
+  maxZoom: number;
+  attribution: string;
+}
+
+interface WmsOptions {
+  enabled: boolean;
+  url: string;
+  selectedTmsLayer: TmsLayer;
+  options: {
+    attribution: string;
+    format: string;
+    layers: string;
+    styles: string;
+    version: string;
+  };
+}
+
+interface WmsInternalOptions {
+  wms: WmsOptions;
+  setValue: VisOptionsSetValue;
+}
+
+function WmsInternalOptions({ wms, setValue }: WmsInternalOptions) {
```

Comment:
You've got a misspelling in the name of this file:

`wms_interna_options.tsx` --> `wms_internal_options.tsx`

Reply:
Thanks 😉